### DISCUSSION TITLE
How to delete or un-publish a shared script ?

### DISCUSSION THREAD

**Discussion by stevebarakat** [johnlindquist/kit#447] [Q&A] [answered]
On Tuesday, I shared a script called "React Sandbox" but later discovered it still had bugs. I wanted to delete it, to prevent it from being published on the next build. Is that possible? I only see an "edit" option.

**Comment by johnlindquist** [comment, marked as answer]
Do you see the labels UI? (It might just be me as a maintainer). Anyway, I'll add a "hide" label to mark it as a post that shouldn't be published.

![CleanShot 2021-09-23 at 16 09 51](https://user-images.githubusercontent.com/36073/134590733-c025ef12-5e39-440d-93dc-57c0cf6aa299.png)

**Reply by stevebarakat** [reply]
I see the labels section but it's missing the gears icon.
![github](https://user-images.githubusercontent.com/3830326/134617556-e67837ab-9602-4408-b3bc-1a7693a7885f.png)

**Reply by johnlindquist** [reply]
Looks like the api doesn't allow me to access labels, so that's not going to work. I'll figure out another way and get back to you soon.
